nrf_rpc/include/nrf_rpc.h

```diff
@@ -127,6 +145,8 @@ struct nrf_rpc_group {
 	void *ack_handler_data;
 	const char *strid;
 	nrf_rpc_err_handler_t err_handler;
+	nrf_rpc_group_bound_handler_t bound_handler;
+	uint32_t flags;
```

Comment:
Does it make sense to make flags const? Seems that they are only set on initialization?

Reply:
I guess it could make sense, but all uses of `nrf_group_t` are `const` to store them in NVM.

Reply:
Done